packages/core/src/providers/atlassian.ts

```diff
@@ -0,0 +1,137 @@
+/**
+ * <div style={{display: "flex", justifyContent: "space-between", alignItems: "center"}}>
+ * <span style={{fontSize: "1.35rem" }}>
+ *  Built-in sign in with <b>Atlassian</b> integration.
+ * </span>
+ * <a href="https://www.atlassian.com/" style={{backgroundColor: "black", padding: "12px", borderRadius: "100%" }}>
+ *   <img style={{display: "block"}} src="https://authjs.dev/img/providers/atlassian.svg" width="24" style={{ marginTop: "-3px"}} />
+ * </a>
+ * </div>
+ *
+ * @module providers/atlassian
+ */
+import type { OAuthConfig, OAuthUserConfig } from "./index.js"
+
+/** The returned user profile from Atlassian when using the profile callback. */
+export interface AtlassianProfile extends Record<string, any> {
+  /**
+   * The user's atlassian account ID
+   */
+  account_id: string
+  /**
+   * The user name
+   */
+  name: string
+  /**
+   * The user's email
+   */
+  email: string
+  /**
+   * The user's profile picture
+   */
+  picture: string
+}
+
+// Any config required that isn't part of the `OAuthUserConfig` spec should belong here
+// For example, we must pass a `redirectUri` to the Atlassian API when requesting tokens, therefore we add it here
+interface AdditionalConfig {
+  redirectUri: string | undefined
+}
+
+/**
+ * ### Setup
+ *
+ * #### Callback URL
+ * ```
+ * https://example.com/api/auth/callback/atlassian
+ * ```
+ *
+ * #### Configuration
+ *
+ * Import the provider and configure it in your **Auth.js** initialization file:
+ *
+ * ```ts title="pages/api/auth/[...nextauth].ts"
+ * import NextAuth from "next-auth"
+ * import AtlassianProvider from "next-auth/providers/atlassian"
+ *
+ * export default NextAuth({
+ *   providers: [
+ *     AtlassianProvider({
+ *       clientId: process.env.ATLASSIAN_ID,
+ *       clientSecret: process.env.ATLASSIAN_SECRET,
+ *       redirectUri: process.env.ATLASSIAN_REDIRECT_URI,
+ *     }),
+ *   ],
+ * })
+ * ```
+ *
+ * ### Configuring Atlassian
+ *
+ * Follow these steps:
+ *
+ * 1. From any page on [developer.atlassian.com](https://developer.atlassian.com), select your profile icon in the top-right corner, and from the dropdown, select **Developer console**.
+ * 2. Select your app from the list (or create one if you don't already have one)
+ * 3. Select **Authorization** in the left menu
+ * 4. Next to OAuth 2.0 (3LO), select **Configure** (or **Add** for newly created app)
+ * 5. Enter the **Callback URL**: `https://{YOUR_DOMAIN}/api/auth/callback/atlassian`
+ * 6. Click Save changes
+ * 7. Select **Settings** in the left menu
+ * 8. Access and copy your app's **Client ID** and **Secret**
+ *
+ * Then, create a `.env` file in the project root add the following entries:
+ *
+ * ```
+ * ATLASSIAN_ID=<Client ID copied in step 8>
+ * ATLASSIAN_SECRET=<Secret copied in step 8>
```

Comment:
```suggestion
 * ATLASSIAN_AUTH_ID=<Client ID copied in step 8>
 * ATLASSIAN_AUTH_SECRET=<Secret copied in step 8>
```

Reply:
Did you mean `AUTH_ATLASSIAN_ID` and `AUTH_ATLASSIAN_SECRET` (as those are what is mentioned for the env file, and also that convention is followed for other providers as well)

I changed them to `AUTH_ATLASSIAN_ID` and `AUTH_ATLASSIAN_SECRET`

Reply:
Yes, sorry